tests/View/Mix/MixTest.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Illuminate\Tests\View\Mix;
+
+use Mockery as m;
+use PHPUnit\Framework\TestCase;
+use Illuminate\Support\HtmlString;
+use Illuminate\View\Mix\MixException;
+
+class MixTest extends TestCase
+{
+    public function setUp()
+    {
+        app()->singleton('path.public', function () {
+            return __DIR__.'/stubs/';
+        });
+    }
+
+    public function tearDown()
+    {
+        m::close();
+    }
+
+    public function testMixMethodWhenDisabled()
+    {
+        $this->assertEquals(new HTMLString('Mix is disabled!'), $this->getMix()->disable()->mix('foo.css'));
+    }
+
+    private function getMix()
+    {
+        return new \Illuminate\View\Mix\Mix();
+    }
+
+    public function testMixMethodThrowAnExceptionIfMixManifestDoesNotExist()
+    {
+        $this->expectException(MixException::class);
```

Comment:
Right, I'm seeing it. I prefer the other way (more consistent imho), but no problem with that.